apps/dashboard/src/lib/ip.ts

```diff
@@ -0,0 +1,9 @@
+import type { NextApiRequest } from "next";
```

Comment:
add `import "server-only";` on the top here so that it *can* only ever be used on the server (and if you try to import it on client it breaks)